### PR TITLE
support vals on WebGPU, run more tests

### PR DESCRIPTION
**Pull request by wpmed92**
- support `vals` using uniforms, test_symbolic passes
- brought back `render_store` because the explicit cast is needed in some cases (example: assigning a cmplt bool result to a i32 output buffer)

### CODE REVIEW

**Comment by chenyuxyz** [comment]
cool! you can remove the `@unittest.skipIf(Device.DEFAULT in ["WEBGPU"], f"{Device.DEFAULT} is not supported")` in test_symbolic_ops.py too

8 minutes is too long for CI, can you disable some slow ones that is not too webgpu specific?

File: test/test_nn.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
 pytestmark = [pytest.mark.exclude_cuda]
 
 class TestNN(unittest.TestCase):
+  @unittest.skipIf(Device.DEFAULT == "WEBGPU", "no int64 on WebGPU")
```
Comment (geohot) [inline test/test_nn.py:14]: Can we just change the test to use torch.int32?

Comment (wpmed92) [reply test/test_nn.py:14]: nn.crossEntropyLoss expects target as Long:

```
 return torch._C._nn.cross_entropy_loss(input, target, weight, _Reduction.get_enum(reduction), ignore_index, label_smoothing)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: expected scalar type Long but found Int
```

**Comment by wpmed92** [comment]
@chenyuxyz Added back `test_symbolic_ops.py` and removed slow tests, now down to 4m.

**Comment by wpmed92** [comment]
@geohot as @chenyuxyz pointed out tests were running too slow so I'm keeping these
`test/test_ops.py test/test_dtype.py test/test_jit.py test/test_symbolic_ops.py test/test_symbolic_jit.py test/test_linearizer.py test/test_linearizer_failures.py test/test_nn.py`

Also, moving back llama compile to webgpu runner otherwise metal runner will be slow.

**Comment by geohot** [comment]
I liked the blacklist better than the whitelist to have new tests be run, but I think this PR is good as is for now.

**Pull request merged**